server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/CloudFormationNodeVisitor.java

```diff
@@ -182,6 +174,10 @@ public void visit(Apache node) {
             Operation configure = node.getStandardLifecycle().getConfigure().get();
             handleOperation(configure, serverName, CONFIG_CONFIGURE);
         }
+        //we add restart apache2 command to the configscript
```

Comment:
Is this always needed?

Reply:
No its not always needed, but there is no problem in restarting it just before the deployment is finished.
If there are modules installed that apache2 correlates with (like php) we need a restart.